3.3.0/scala2.12-java11-python3-r-ubuntu/Dockerfile

```diff
@@ -24,9 +24,9 @@ RUN groupadd --system --gid=${spark_uid} spark && \
 RUN set -ex && \
     apt-get update && \
     ln -s /lib /lib64 && \
-    apt install -y gnupg2 wget bash tini libc6 libpam-modules krb5-user libnss3 procps net-tools gosu && \
-    apt install -y python3 python3-pip && \
-    apt install -y r-base r-base-dev && \
+    apt-get install -y gnupg2 wget bash tini libc6 libpam-modules krb5-user libnss3 procps net-tools gosu && \
```

Comment:
this should be changed, we apply this change only after 3.4 version.

It would be good if you can revert all 3.3.0 changes, : )